flake.nix

```diff
@@ -12,7 +12,7 @@
   #   $ nix run github:GaloisInc/flakes#z3.v4_8_14 -- -version
 
   inputs = {
-    nixpkgs.url = "github:nixos/nixpkgs/22.05";
+    nixpkgs.url = "github:nixos/nixpkgs/23.05";
```

Comment:
I needed to make this change to gain access to `meson >= 0.64`, which `bitwuzla` requires.